src/models/chan.js

```diff
@@ -23,18 +23,21 @@ function Chan(attr) {
 	}, attr));
 }
 
-Chan.prototype.sortUsers = function() {
-	this.users = _.sortBy(
-		this.users,
-		function(u) { return u.name.toLowerCase(); }
-	);
-
-	["+", "%", "@", "&", "~"].forEach(function(mode) {
-		this.users = _.remove(
-			this.users,
-			function(u) { return u.mode === mode; }
-		).concat(this.users);
-	}, this);
+Chan.prototype.sortUsers = function(irc) {
+	var userModeSortPriority = {};
+	irc.network.options.PREFIX.forEach(function(prefix, index) {
```

Comment:
We probably can change `toJSON` method in network to return whitelisted properties, like `export` does.